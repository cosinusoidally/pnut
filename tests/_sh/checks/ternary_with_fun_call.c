// expect_comp_failure
void main() {
  int a = f() ? 1 : 2; // Valid
  int b = f() ? 1 : f(); // Invalid
}
