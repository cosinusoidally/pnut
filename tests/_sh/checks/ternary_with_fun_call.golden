tests/_sh/checks/ternary_with_fun_call.c:6:0  Conditional function calls in ternary operator not allowed
